CRM/Pledge/BAO/Query.php

```diff
@@ -524,7 +524,7 @@ public static function getPledgePaymentSearchFieldMetadata() {
   }
 
   /**
-   * Build the search for for pledges.
+   * Build the search for pledges.
```

Comment:
Actually I think it was supposed to read "Build the search **form** for pledges."

But it's a pretty useless comment if you ask me. The function is named `CRM_Pledge_BAO_Query::buildSearchForm`, so yea it's already quite obvious that it's meant to build a search form for pledges.

IMO if a comment doesn't add any useful information then it's just noise. What would be much more useful is a comment saying what type of form this is building and in what context this is used. Is this used to build an advanced search pane? A standalone search form? Both?

Reply:
@colemanw the drupal standard is there should be a short sentence overview (beginning with a Capital & ending with a full stop) - regardless of whether it adds much info - but more info is better